src/main/kotlin/uk/gov/gdx/datashare/models/CreateAcquirerRequest.kt

```diff
@@ -1,10 +1,23 @@
 package uk.gov.gdx.datashare.models
 
 import io.swagger.v3.oas.annotations.media.Schema
+import org.hibernate.validator.constraints.Length
 import uk.gov.gdx.datashare.enums.EventType
+import javax.validation.constraints.Pattern
 
-data class CreateAcquirerRequest(
-  @Schema(description = "Name of client, must be lower case", required = true, example = "HMPO")
+class CreateAcquirerRequest(
+  @Schema(
+    description = "Name of client, may only contain lowercase letters, numbers, and the following special characters: + = , . @ -",
+    required = true,
+    example = "hmpo",
+    maxLength = 128,
+    pattern = "^[a-z0-9+=,.@-]*\$",
+  )
+  @get:Length(min = 3, max = 128)
```

Comment:
Is it worth writing _somewhere_ why these constraints exist? E.g. if we moved off of Cognito, would these change?